tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -1,80 +1,136 @@
 from __future__ import division
 
+import mock
+import numpy as np
+import random
+import tempfile
 import unittest
 
+from chainer import serializers
 from chainer import testing
 from chainer import training
 
 
-class DummyUpdater(training.Updater):
+def get_trainer_with_mock_updater(iter_per_epoch):
```

Comment:
I noticed that we can change the trigger used in `get_trainer_with_mock_updater` by `stop_trigger` argument.